### PR TITLE
[FAB-17189] Add unit test for statecouchdb getRevisions.

### PR DESCRIPTION
**Pull request by DereckLuo**
Signed-off-by: Chongxin Luo <Chongxin.Luo@ibm.com>

#### Type of change

- Improvement (improvement to code, performance, etc)
- Test update

#### Description

- Added _commit_handling_test.go_ file.
- Added unit test for statecouchdb package getRevisions.
#### Related issues

[FAB-17189](https://jira.hyperledger.org/browse/FAB-17189)

### CODE REVIEW

**Comment by DereckLuo** [comment]
simplify test to two function calls on _getRevisions_, and combine all three test cases into one single _nsUpdates_.

**Review by manish-sethi — COMMENTED** [review]
Also, as we discussed, Can you club the data for the green path such that some of the data is present in the cache, some in the db, some in both and some nowhere.

File: core/ledger/kvledger/txmgmt/statedb/statecouchdb/commit_handling_test.go [hunk 1]
```diff
@@ -0,0 +1,68 @@
+/*
+Copyright IBM Corp. All Rights Reserved.
+SPDX-License-Identifier: Apache-2.0
```
[the remark this answers is not in the document]
Comment (DereckLuo) [reply core/ledger/kvledger/txmgmt/statedb/statecouchdb/commit_handling_test.go:4]: should I fix this for rest of the code base as well? or push another PR for this is more appropriate?

**Comment by DereckLuo** [comment]
@manish-sethi thanks for the review. 
- test data has been squashed into one single struct. Following green paths are tested.
   - data present in cache only
   - data present in both cache and db
   - data present in db only
   - data does not present in neither cache nor db.
- fixed license header for this file.

File: core/ledger/kvledger/txmgmt/statedb/statecouchdb/commit_handling_test.go [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+			Metadata: nil,
+			Version:  version.NewHeight(1, 5),
+		},
+	}
+
+	revisionsMap, err := db.getRevisions("ns", nsUpdates)
+	assert.NoError(t, err)
+	assert.Equal(t, "revision-cache-number", revisionsMap["key-in-cache"])
+	assert.Equal(t, "revision-db-number", revisionsMap["key-in-both-db-cache"])
+	assert.Equal(t, "", revisionsMap["bad-key"])
```
Comment (manish-sethi) [inline core/ledger/kvledger/txmgmt/statedb/statecouchdb/commit_handling_test.go:75]: Can add one more assertion
assert.NotEqual(t, "", revisionsMap["key-in-db"])

File: core/ledger/kvledger/txmgmt/statedb/statecouchdb/commit_handling_test.go (hunk 1 above)
[the remark this answers is not in the document]
Comment (manish-sethi) [reply core/ledger/kvledger/txmgmt/statedb/statecouchdb/commit_handling_test.go:4]: I have been doing as I fix something else in these files. However, if you want to fix this in on shot, I would suggest a dedicated PR.

**Review by manish-sethi — COMMENTED** [review]
LGTM, a minor comment

**Comment by DereckLuo** [comment]
added the additional assertion, and I'll open a separate PR to fix license layouts. Thanks.

**Review by manish-sethi — APPROVED** [review]

**Pull request merged**